client/src/components/layouts/Learn.js

```diff
@@ -41,6 +41,13 @@ class LearnLayout extends Component {
     this.props.tryToShowDonationModal();
   }
 
+  componentWillUnmount() {
+    const metaTag = document.querySelector(`meta[name="googlebot"]`);
```

Comment:
```suggestion
    const metaTag = document.querySelector(`meta[name="robots"]`);
```
`robots` is more generic and includes `googlebot`

Reply:
this needs to be changed on two different lines. Pushing a commit with the complete changes.